lib/terraform.nix

```diff
@@ -11,19 +11,26 @@ in
 , tfConfig ? null
 , subdir ? defaultSubDir
 , backend ? true
-, vault-addr ? "https://vault.serokell.org:8200"
 }:
 rec {
   mkApp = let
     defaultSubDir = subdir;
     defaultBackend = backend;
     defaultTFConfig = tfConfig;
+
+    # Some of our configurations use the VPC module, which is sourced from the nix store
+    # So we have to explicitly remove context with builtins.unsafeDiscardStringContext
+    tfConfigNoContext = builtins.fromJSON (builtins.unsafeDiscardStringContext tfConfig.value);
```

Comment:
Hmm, but can't you pass a terranix configuration in a state before JSON to avoid these unsafe conversions? I.e. `tfConfig` would be an attrset instead of JSON file

Reply:
oh, indeed we can do this using [`terraformConfigurationAst`](https://github.com/terranix/terranix/blob/ac98723a6c66fce3f91b0e8cbf8911430736f130/flake.nix#L107) instead of `terraformConfiguration`.